packages/lndr/data.ts

```diff
@@ -0,0 +1,245 @@
+export default [ 
```

Comment:
can you name this file something like `countries.ts`? Also, please add in a field called `code` that is the 3-letter country code

Reply:
Great! I'll effect the changes.